internal/providers/terraform/aws/elastic_beanstalk_environment.go

```diff
@@ -0,0 +1,97 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/resources/aws"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+func getElasticBeanstalkEnvironmentRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_elastic_beanstalk_environment",
+		RFunc: newElasticBeanstalkEnvironment,
+	}
+}
+
+func newElasticBeanstalkEnvironment(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+
+	var region = d.Get("region").String()
+	var StreamLogs = false
+	var DBIncluded = false
```

Comment:
```suggestion
	var streamLogs = false
	var dbIncluded = false
```

Reply:
Capitalised variables in GO denote exported types https://www.ardanlabs.com/blog/2014/03/exportedunexported-identifiers-in-go.html